Correct relative clause part-of-speech disagreement (#2314)
microsoft/TypeScript-Website: packages/documentation/copy/en/release-notes/TypeScript 2.0.md

## Read-only properties and index signatures

> Now a property or index signature declared with the `readonly` modifier is considered read-only.

Reviewer: If anything, I'd say:

```suggestion
A property or index signature can now be declared with the `readonly` modifier.
```

Because this section is only aiming to introduce the feature; the description of what it does immediately follows in the next paragraph.
